### DISCUSSION TITLE
I can't update

### DISCUSSION THREAD

**Discussion by 849025143** [immich-app/immich#13533] [Q&A] [answered]
### The bug

I use 1.115. I haven't received any update information, and updating with Docker doesn't work either.

### The OS that Immich Server is running on

Debian

### Version of Immich Server

1.115

### Version of Immich Mobile App

Laster

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
default
```

### Your .env content

```Shell
default
```

### Reproduction steps

Just update

**Comment by scherddel** [comment]
What is the error you recieve when you try to "docker compose pull"? A little bit more of information would help :-)

**Reply by 849025143** [reply]
![1132](https://github.com/user-attachments/assets/10335ea0-d384-4aaf-bf43-f65d152fb02a)
This

**Reply by 849025143** [reply]
hello？

**Comment by tbelway** [comment]
If you need help updating you should probably provide more information.
What errors are you getting? Your screenshot above provides pretty much no info.
What happens if you try to just pull the image?
What errors appear in journal?
What does your yaml look like?

If you're going to provide 0 information about what your problem is, you'll get as many responses...

**Reply by 849025143** [reply]
[env+docker-compose.zip](https://github.com/user-attachments/files/17468138/env%2Bdocker-compose.zip)

oh,okay. file in here

**Reply by tbelway** [reply]
Your config looks ok, assuming the santized elements are also good and that the directories exist as defined. But you also posted an issue that clearly demonstrates a docker configuration issue (or perhaps a network/dns issue).
can you:
```bash
nslookup registry-1.docker.io
```
?

**Comment by 849025143** [comment]
![123555](https://github.com/user-attachments/assets/57bdad37-4c7e-495a-b83d-820721718ded)

**Reply by scherddel** [reply]
what does ```docker info``` print?

It seems your docker is not configured properly.

**Reply by tbelway** [reply, marked as answer]
I agree, ths isn't an immich issue but a docker issue (or perhaps a network/dns issue).
@849025143 can you try pulling a different container, pretty much anything other than immich, and see if it fails as well?

**Reply by 849025143** [reply]
oh,My network seems to have some issues. Now it's fine.
